automated install/basic-install.sh

```diff
@@ -2330,6 +2338,9 @@ main() {
         fi
     fi
 
+    # Check if SELinux is Enforcing
+    checkSelinux
```

Comment:
Should this be after the distro check?

Reply:
Reasoning?

Reply:
If someone is on an unsupported distro with SELinux enabled, they will get the SELinux warning before knowing that their distro is not supported.

Reply:
Fair, I was trying to avoid any repos being enabled, packages installed, or other dialogs. I am fine with moving it behind distro check. The distro _check_ has slowly became more of a distro _config_.

Reply:
That will be rectified soon. We'll have a static list of distributions that we support and the installer will check against that before any actions are taken. Then the script can be a configuration checker.

Reply:
Sounds good, I will move the selinux check behind distro check for now.

Reply:
I just cherry-picked my other two commits and force pushed.  
The selinux check depends on `whiptail` so it must exist after installer deps have been installed.